tests/test_app/components.py

```diff
@@ -44,15 +44,6 @@ def parameterized_component(x, y):
     )
 
 
-victory = web.module_from_template("react", "victory-bar", fallback="...")
-VictoryBar = web.export(victory, "VictoryBar")
-
-
-@component
-def simple_bar_chart():
-    return html._(VictoryBar(), html.hr())
-
-
```

Comment:
We can remove the use of `module_from_template`, however we do need something that tests whether or not web modules load properly. The most reliable way to do that would be to use `module_from_file` and write a super simple JS module. We [do this](https://github.com/idom-team/idom/blob/976028073fc3fba43870fdbeb3af96cad6e77ed1/tests/test_web/js_fixtures/simple-button.js) in IDOM core presently.